<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>artCartoonDialog</class>
 <widget class="QDialog" name="artCartoonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cartoon</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelThreshold">
       <property name="text">
        <string>Threshold</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="ADM_SliderIndicator" name="horizontalSliderThreshold">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelScatter">
       <property name="text">
        <string>Scatter</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ADM_SliderIndicator" name="horizontalSliderScatter">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelLevel">
       <property name="text">
        <string>Color level</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ADM_SliderIndicator" name="horizontalSliderColor">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="toolboxLayout"/>
   </item>
   <item>
    <widget class="ADM_QSlider" name="horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ADM_QSlider</class>
   <extends>QSlider</extends>
   <header>ADM_toolkitQt.h</header>
  </customwidget>
  <customwidget>
   <class>ADM_SliderIndicator</class>
   <extends>QSlider</extends>
   <header>ADM_toolkitQt.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>horizontalSliderThreshold</tabstop>
  <tabstop>horizontalSliderScatter</tabstop>
  <tabstop>horizontalSliderColor</tabstop>
  <tabstop>horizontalSlider</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>artCartoonDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>artCartoonDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
